pkg/client/suite_test.go

```diff
@@ -38,7 +38,7 @@ var (
 	busyboxDest = filepath.Join(busyboxDestDir, "busybox")
 	runtimePath = os.Getenv("RUNTIME_BINARY")
 	conmonPath  = os.Getenv(conmonBinaryKey)
-	maxRSSKB    = 7500
+	maxRSSKB    = 10000
```

Comment:
:(

Reply:
We can start the server on demand to reduce the amount of basic memory used.